docs/reference/Func/SECValidateAccessToken.md

```diff
@@ -0,0 +1,67 @@
+##### Function : User Registration
+##### SECValidateAccessToken - Validates a signed JWT access token.
+---
+```
+#include <bsafe.h>
+STATUS LNPUBLIC  SECValidateAccessToken(
+
+	const char  *pszAccessToken,
+	const char  *pszProviderURL,
+	const char  *pszRequiredScope,
+	const char  *pszResourceURL,
+	DWORD  dwFlags,
+	void  *vpOptionalParams,
+	DWORD  dwMaxEmailSize,
+	char  *retszEmail,
+	DWORD  *retdwDurationSec);
+```
+
+**Description :**
+
+Validates a signed JWT access token that was generated by a trusted OIDC provider. Intended to be called by multiple different server tasks, including external code.
```

Comment:
What is JWT & OIDC stands for ?

Reply:
JWT - JSON Web Token
OIDC - Open ID Connect